pkgs/cosmic-ext-applet-system-monitor/package.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  fetchFromGitHub,
+  libcosmicAppHook,
+  rustPlatform,
+  just,
+  stdenv,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "cosmic-ext-applet-system-monitor";
+  version = "0.1.2";
```

```suggestion
  version = "0-unstable-2025-01-13";
```

Not sure where this version comes from, but I don't see a tag upstream so let's use what the auto-updater will use